server/game/cards/01_SOR/leaders/EmperorPalpatineGalacticRuler.ts

```diff
@@ -0,0 +1,69 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { LeaderUnitCard } from '../../../core/card/LeaderUnitCard';
+import { RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class EmperorPalpatineGalacticRuler extends LeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '5784497124',
+            internalName: 'emperor-palpatine#galactic-ruler',
+        };
+    }
+
+    protected override setupLeaderSideAbilities() {
+        this.addActionAbility({
+            title: 'Deal 1 damage to a unit and draw a card',
```

Comment:
Update to "Defeat a friendly unit to deal 1 damage to a unit and draw a card"

Reply:
How many cards with non-resource costs have we implemented? It's not a must-have but I think it's worth considering